src/pyuvsim/simsetup.py

```diff
@@ -2145,13 +2139,6 @@ def _complete_uvdata(uv_in, inplace=False, check_kw=None):
     uv_obj.flag_array = np.zeros(_shape, dtype=bool)
     uv_obj.nsample_array = np.ones(_shape, dtype=float)
 
-    uv_obj.extra_keywords = {}
-
-    if check_kw is not False:
-        if check_kw is None:
-            check_kw = {}
-        uv_obj.check(**check_kw)
-
```

Comment:
I don't think there is any way that the things done in this function can cause checks to fail if the input object is ok. Since this is a private method, we don't need to guarantee behavior for random users, only ourselves, so I think this is unnecessary.